webpack.config.js

```diff
@@ -28,15 +28,18 @@ const config = {
       path.resolve('./node_modules'),
     ],
   },
+  devtool: 'eval-source-map',
   module: {
+    // We need to load flexboxgrid with css-modules, but others need to be loaded
+    // with regular css loader.
     loaders: [{
       test: /\.css$/,
       loader: 'style-loader!css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader',
-      exclude: /node_modules/,
+      exclude: /node_modules\/?=flexboxgrid/,
```

Comment:
yea, I'm not sure if there is a better way to do this, but we can engineer something if it ends up becoming complex.